config/application.yml.example

```diff
@@ -4,6 +4,7 @@
 # Minimum 30 but usually 128 characters. To obtain run 'rake secret', or faster, 'openssl rand -hex 128'
 SECRET_TOKEN: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 
+# Time zone must match the operating system time zone in order to pass all tests. This is a string which is the key for the MAPPING hash constant in the ActiveSupport::TimeZone class. Documentation for this class including the entire hash can be found here: https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html
```

Comment:
Could you please wrap this long line. It's hard to read?

Reply:
Good point. Changes submitted.